src/components/Dialog/ProductTransferDialog.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-dialog
+    :model-value="modelValue"
+    @update:model-value="modelValue"
+    persistent
+    @click:outside="closeDialog"
+  >
+    <v-form @submit.prevent="handleSubmit" ref="form" class="mt-4">
+      <v-card class="mx-auto text-center pa-2" max-width="400" min-width="200">
+        <v-card-title>
+          Catalog Number: {{ product.catalogNumber }}
+        </v-card-title>
+        <v-select
+          :items="userOptions"
+          v-model="user"
+          label="Select User"
+          required
+          :rules="[validateUser]"
+        ></v-select>
+        <div class="d-flex mx-auto">
+          <v-btn color="success" class="mt-4" type="submit">Submit</v-btn>
+          <v-btn color="error" class="mt-4" @click="closeDialog">Close</v-btn>
+        </div>
+      </v-card>
+    </v-form>
+  </v-dialog>
+</template>
+
+<script setup>
+import { validateUser } from "@/utils/validation-rules";
+import { ref, computed, inject } from "vue";
+import { useStore } from "vuex";
+const snackbarProvider = inject("snackbarProvider");
+const props = defineProps(["modelValue", "product", "userId"]);
```

Comment:
Let's keep the way we defineProps consistent. We are defining them differently here compared to the other components:
`const props = defineProps(["modelValue", "product", "userId"]);`
vs
```
const props = defineProps({
modelValue: Boolean, 
product: Object,
userId: String
});
```
Is there a reason for this change here?

Reply:
updated